source/reference/connection-string.txt

```diff
@@ -323,12 +323,74 @@ Connection Options
        timeout, though different drivers might vary. See the
        :ecosystem:`driver </drivers>` documentation.
 
+   * - .. urioption:: compressors
+
+     - Comma-delimited string of compressors to enable
+       network compression for communication between this
+       client and:
+
+       - a :binary:`~bin.mongod` instance
+
+       - a :binary:`~bin.mongos` instance.
+
+       You can specify the following compressors:	
+
+       - :term:`snappy`
+
+       - :term:`zlib` (Available in MongoDB 3.6 or greater)
+
+       If you specify multiple compressors, then the order in which you list
+       the compressors matter as well as the communication initiator. For
+       example, if the client specifies the following network
+       compressors ``zlib,snappy`` and the :binary:`~bin.mongod` specifies
```

Comment:
From looking at the rendered view, adding quotes to the compressors list examples (e.g. ` ``"zlib,snappy"`` `) might make them more readable in the browser, as it's a little tricky to notice that the comma before "messages" is not part of the compressors list. This is totally outside my expertise, though, so feel free to ignore this if you disagree!

Reply:
Thank you!  Will definitely add the quotes 👍